Xamarin.Forms.Controls/GalleryPages/SwipeViewGalleries/SwipeThresholdGallery.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.GalleryPages.SwipeViewGalleries
+{
+	[Preserve(AllMembers = true)]
+	public class SwipeThresholdGallery : ContentPage
+	{
+		public SwipeThresholdGallery()
```

Comment:
Android/ios: The threshold doesn't quite seem right. On iOS if I set the slider to the middle or all the way to the right the action will get invoked if I only swipe to the middle.  

![B93D3AAC-6E34-406D-8BA2-390040C73ADD](https://user-images.githubusercontent.com/5375137/65463678-6c0d4800-de15-11e9-818c-33f468f4d1e9.GIF)


UWP: If I set the threshold all the way to the left then the view still swipes all the way open before executing 


![B176A920-8255-4960-B1C9-9B0A7630593B](https://user-images.githubusercontent.com/5375137/65463602-44b67b00-de15-11e9-905a-b2c78d015d23.GIF)

Reply:
The SwipeThreshold property is not in the Spec and I have removed it (at least for now).